aio_pika/channel.py

```diff
@@ -56,158 +55,180 @@ def __init__(
 
         self.loop = connection.loop
 
-        self._connection = connection
-        self._done_callbacks = CallbackCollection(self)
-        self._return_callbacks = CallbackCollection(self)
-        self._channel = None  # type: Optional[aiormq.Channel]
+        self._channel: aiormq.abc.AbstractChannel
         self._channel_number = channel_number
+
+        self.connection = connection
+        self.close_callbacks = CallbackCollection(self)
```

Comment:
Why not private?

Reply:
I want to declare management methods (e.g. `add_close_callback`) deprecated since 8.0